### ISSUE TITLE
Use SVG icons from Ameixa

### ISSUE THREAD

**Issue by conrad-heimbold** [dkanada/ICEcons#64]
@dkanada @ikocevski I just noticed that the [Ameixa CM/LOS icon pack](https://gitlab.com/xphnx/twelf_cm12_theme) by [Xphnx](https://gitlab.com/xphnx) already has [lots of SVG icons](https://gitlab.com/xphnx/twelf_cm12_theme/tree/master/svgs/depot). Unfortunately, there are quite bloated (every icon more than 60 KB, of which only 2KB are needed); so **you can't extract the relevant icon path easily**. I spent some time on that (lots of steps necessary...) and I'm almost finished. 

**Please have a look on the [SVG files in the attached ZIP file](https://github.com/dkanada/ICEcons/files/1166469/ameixa_optim.zip)!**

We could easily use them by copy-pasting them onto the circle/square/rectangle template and substracting (This could also be automated).

**Comment by ikocevski** [comment]
I don't think we can automate this. Too many gimmicks here and there. So unclean. Tried making the FFUpdater. Besides, we should have our own style. A line or two makes a world of a difference.

**Comment by conrad-heimbold** [comment]
Hi, I managed to finish - **for 499 out of 656 svg icons, my script worked!!!**. **The attached 499 svg icons only consist of one path**, nothing else: [ameixa_optim_finished.zip](https://github.com/dkanada/ICEcons/files/1167405/ameixa_optim_finished.zip)

To get more icons for us.. 
1. **Copy-Paste** one of the attached SVG files **onto a suitable template** from us 
(CTRL+A CTRL+C CTRL+V) 
2. **Resize** it to a suitable size. 
3. **Subtract** the path from the template (CTRL+MINUS) 

=> One more icon finished! 
This takes only around 1 minute per icon, so we should get these icons finished very fast.

**Comment by ikocevski** [comment]
Well, good job. I guess some of these will be useful.

**Comment by dkanada** [comment]
Woah, mind sharing that script? It sounds interesting. I am on vacation for a while but when I have time I will take a look at these.

**Comment by conrad-heimbold** [comment]
I developed that script by trial-and-error, in many steps and in a very customized version just for these specific files and without saving the commands in a file (except of ``~/.bash_history``). So getting the commands again might be difficult... Let's see if I can find them again in my ``~/.bash_history`` ...

**Comment by conrad-heimbold** [comment]
In the script, I used roundabout the following: 
- svgo with lots of options (--enable=... )
- tr "delete all [:space:]" 
- sed "delete everything between <defs> and </defs>" 
- sed "delete **first** group (everything between <g> and </g>) that should **not** contain: 
```
transform="translate(0 -1025)"
```
(this is specific for the group of objects that we need)
- sed "delete all elements containing fill=none"

**Comment by conrad-heimbold** [comment]
I think you should be able to re-construct a similarly useful script with these tips. My script was very bad (I don't know how to use awk / perl / python, which would have been a better choice than sed).
